src/modules/drive/Toolbar/delete/DeleteItem.jsx

```diff
@@ -11,19 +11,30 @@ import { translate } from 'cozy-ui/transpiled/react/providers/I18n'
 
 import deleteContainer from './delete'
 
+import { SHARED_DRIVES_DIR_ID } from '@/constants/config'
+
 const DeleteItem = ({ t, isSharedWithMe, trashFolder, displayedFolder }) => {
   const handleClick = () => {
     trashFolder(displayedFolder)
   }
 
-  const label = isSharedWithMe ? t('toolbar.leave') : t('toolbar.trash')
+  const isSharedDrive = useMemo(
+    () => displayedFolder.dir_id === SHARED_DRIVES_DIR_ID,
+    [displayedFolder]
+  )
```

Comment:
Why did you use `useMemo` ? It is necessary ? Looks good otherwise 👍

Reply:
you're right, we don't need useMemo here